src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -38,6 +37,11 @@ public class ConnectionService
 
         public const int MaxServerlessReconnectTries = 5; // Max number of tries to wait for a serverless database to start up when its paused before giving up.
 
+        // SQL Error Code Constants
+        private const int DoesNotMeetPWReqs = 18466;

```

Comment:
Can you please add comment here with error message or link to error info so it's easy to refer from.